tests/e2e/contacts/new-lineage-structure.wdio-spec.js

```diff
@@ -21,7 +21,7 @@ describe('Create new lineage structure', () => {
   });
 
   after(async () => {
-    await utils.revertDb([], true);
+    await utils.revertDb([], 'api');
```

Comment:
If I'm looking at the right utils, the second parameter is always treated as a boolean: https://github.com/medic/cht-core/blob/8bdce5bdc3731814e06aa113154ae1792b5a2323/tests/utils.js#L258

What am I missing?

Reply:
You're right! I'll revert. Only `updateSettings` passes the parameter along. I'll revert. Thanks!